html-hello-world/package.json

```diff
@@ -2,8 +2,8 @@
     "scripts": {
         "postinstall": "spago install",
         "bundle-ps": "spago bundle-module --to ./output/bundle.js",
-        "build": "npm bundle-ps && parcel build index.html",
-        "start": "npm bundle-ps && concurrently \"spago bundle-module --to ./output/bundle.js --watch\" \"parcel index.html\""
+        "build": "npm run bundle-ps && parcel build index.html",
+        "start": "npm run bundle-ps && concurrently \"spago bundle-module --to ./output/bundle.js --watch\" \"parcel index.html\""
```

Comment:
Yep, minor but annoying difference to `yarn`. Probably more future proof for `npm` though 😄